pymc3/step_methods/metropolis.py

```diff
@@ -811,6 +876,306 @@ def competence(var, has_grad):
         return Competence.COMPATIBLE
 
 
+class MLDA(ArrayStepShared):
```

Comment:
I think this should live in its own `mlda.py` file.

Reply:
> Also, @gmingas, when addressing the review comments, can you do another rebase off upstream/master please?
> Sorry to ask you that again, but we fixed the test environment in a recent PR, so we need this rebase for the tests to pass again on this PR

Will do, no worries. I noticed there were a few issues with Travis.

Reply:
Move to mlda.py and changes also made to step_methods/__init__.py